src/Kafka/Type.purs

```diff
@@ -25,6 +25,18 @@ type MessageHeaders =
 type MessageHeadersImpl =
   Foreign.Object.Object String
 
+-- | * `isr`
+-- |   * in-sync replicas
```

Comment:
Thanks for adding the comments. This is an improvement, though I think it would be even better just to have the label be `inSyncReplicas`. It would also be in line with our style guide rule about using [clear names](https://github.com/citizennet/orders/blob/main/docs/reference/style-guide-and-best-practices.md#clear-names).

Reply:
`isr` is official terminology in Kafka. If you follow [docs/shell-commands](https://github.com/citizennet/purescript-kafkajs/blob/main/docs/shell-commands.md) and run `kafka-topics --describe`, `isr` is in the result like

```
Topic: test1	PartitionCount: 3	ReplicationFactor: 3	Configs: 
	Topic: test1	Partition: 0	Leader: 2	Replicas: 2,1,0		Isr: 2,1,0
	Topic: test1	Partition: 1	Leader: 1	Replicas: 1,0,2		Isr: 1,0,2
	Topic: test1	Partition: 2	Leader: 0	Replicas: 0,2,1		Isr: 0,2,1
```

Reply:
Ah I see. I still think `isSyncReplicas` would be clearer, but if `isr` is part of the Kafka domain terminology and users of the library can be expected to know that, I guess it's okay.